pkg/userdata/ubuntu/provider.go

```diff
@@ -271,6 +273,7 @@ write_files:
 {{- /* If something failed during package installation but docker got installed, we need to put it on hold */}}
     apt-mark hold docker.io || true
     apt-mark hold docker-ce || true
+    apt-mark hold docker-ce-cli || true
```

Comment:
This is not a blocker but I would have done it in one single line:

```
apt-mark hold docker-ce docker-ce-cli || true
```

Reply:
That's a good point, but let's leave it for a follow-up along with `CR_PKG`. Tests are already running and I think we should try to get this merged as soon as possible.